RepoDb.Core/RepoDb/Extensions/DbConnectionExtension.cs

```diff
@@ -3083,7 +3083,8 @@ internal static IEnumerable<Field> GetQualifiedFields<TEntity>(IEnumerable<Field
             var items = values is IEnumerable<object> ? (IEnumerable<object>)values : values.WithType<object>();
             if (items.Any() != true)
             {
-                return commandText;
+                var parameter = parameterName.AsParameter(dbSetting);
+                return commandText.Replace(parameter, string.Concat("(select ", parameter, " where 1 = 0)"));
```

Comment:
Just a small comment, let us make all the SQL statement UPPERCASE like `SELECT * FROM Table WHERE 1 = 0;`.